lib/db.ts

```diff
@@ -0,0 +1,6 @@
+import { Pool, Client } from 'pg';
+const connectionString = process.env.DATABASE_URL;
+
+const dbClient = new Client({connectionString});
+
+export default dbClient;
```

Comment:
Are we using this `export` somewhere in the repo? I couldn't find any reference to it in the code.

If we are not using the file, then I guess we could:
- delete current `db.ts`
- rename `client.ts` to `db.ts`
- adjust `imports` to use `db.ts` instead of `client.ts`

Maybe it is just me, but `client.ts` sounds too generic. It could mean a cache, db or queue clients, for example. On the other hand, if we call it `db.ts` its name will hint devs about the intent of the script.